src/server/user.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import Header from './components/header';
+
+const { userInfo, thisUser, notebookList } = window;
+
+render(
+  <div>
+    <Header userInfo={userInfo} />
+    <div style={{
+        marginLeft: '30px',
+        marginTop: 35,
+    }}
+    >
+      <div style={{ marginBottom: 0 }}>
+        <img style={{ borderRadius: '5px' }} width={150} src={thisUser.avatar} alt={`${thisUser.name}'s avatar`} />
+        <h1 style={{
+            marginTop: 20, marginBottom: 0, fontWeight: 900, textTransform: 'uppercase',
+            }}
+        >{thisUser.full_name}
+        </h1>
+        <h2 style={{ fontWeight: 300, marginTop: 0 }}>{thisUser.name}</h2>
+      </div>
+      <h2>notebooks</h2>
+      <table >
+        <tbody>
+          {notebookList.map(notebook => (
+            <tr key={notebook.id}>
+              <td style={{ width: 200 }}><a href={`/notebooks/${notebook.id}/`}>{notebook.title}</a></td>
+              <td>{notebook.last_revision}</td>
```

Comment:
Could you use something like [moment.js](https://momentjs.com/) to make the time column more readable? We can do this in a follow up ofc.

Reply:
Totally agreed, and also agreed that we can take this on when we're focusing on styling this page.